src/pages/workspace/WorkspaceMoreFeaturesPage.tsx

```diff
@@ -204,6 +204,13 @@ function WorkspaceMoreFeaturesPage({policy, route}: WorkspaceMoreFeaturesPagePro
                     return;
                 }
                 if (isEnabled) {
+                    if (policy?.type !== CONST.POLICY.TYPE.CORPORATE) {
```

Comment:
NAB: could you please replace this with the util function: `isControlPolicy()` https://github.com/Expensify/App/pull/43151/files#diff-89601c1d4ab7d6fb789ea45b8f8e16ead0f925cf0db25d9b5c3561816d7a94c1

The function is currently not in `main`, so you can just copy paste. Hope that's okay : )